### DISCUSSION TITLE
Secret Service Activation

### DISCUSSION THREAD

**Discussion by peerwal** [keepassxreboot/keepassxc#6141] [Q&A] [answered]
Hi everybody, I just installed KeepassXC 2.6.4 Flatpak on my KDE Neon system. It is working fine but I can't find the 'Activate Secret Service Option' anywhere and I an wondering, whether this option is not availabel with the flatpak?

**Comment by droidmonkey** [comment, marked as answer]
https://github.com/flathub/org.keepassxc.KeePassXC/issues/73
